cfy_manager/components/restservice/scripts/create_default_db.sh

```diff
@@ -61,8 +67,16 @@ function create_composer_database() {
     run_psql "ALTER DATABASE $db OWNER TO $user;"
 }
 
+function possibly_revoke_role_from_user() {
+    user=$1
+    if [ -n "$PGUSER" ]; then
```

Comment:
maybe add a debug msg here, if this will fail

Reply:
There is a debug per command in `run_psql` function